pages/common/git-commit.md

```diff
@@ -14,3 +14,7 @@
 - Replace the last commit with currently staged changes:
 
 `git commit --amend`
+
+- Commit only specific files or folders:
+
+`git commit -m {{message}} {{path/to/my/file.ext}}`
```

Comment:
Maybe add two files, or a glob expression?

Also, how does this interact with the staged changes? E.g. you stage file A but commit file B, do both A and B get committed, or just B? Or if you stage part of file B, does this commit the entirety of B or just the staged parts?

Reply:
You cannot commit files that are not staged!

If you stage for example A and B and do `git commit -m "msg" A` then only file A will be commit.

Reply:
I would add second file for example.

Reply:
> You cannot commit files that are not staged!

Well, you can with `commit -a`, so the behavior I described wouldn't be that out of place, IMO.

So what you're saying is that this can serve as a filter for the staged changes? E.g. if I stage file A, file B and half of file C, then `git commit A C` will commit only file a and half of file C, leaving file B staged and ready for another commit?

Reply:
I would specify that files have to be staged though, something like:

> Commit only specific (already staged) files or directories:

Also: directories, not folders :P

Reply:
@waldyrious afaik `-a` flag will just do `add` command before commit. But I agree with you 👍 

@mebeim Probably: `Commit only specific (already staged) files` is good enough?

Reply:
Yeah @Aracki I think that sounds good :)

Reply:
This is clearer, yes. Now if this works with [`commit --patch`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--p), that means I'll have a feature I've been yearning for a while! Multi-level staging :P

Reply:
@Aracki I'm leaving this conversation "unresolved" so it doesn't get collapsed, since it contains important details that may inform future readers.